test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/TagHelperBlockRewriterTest/CreatesErrorForEmptyTagHelperBoundAttributes12.stree.txt

```diff
@@ -1,8 +1,37 @@
-Markup block - Gen<None> - 32 - (0:0,0)
-    Tag block - Gen<TagHelper> - 32 - (0:0,0) - myth - mythTagHelper
-        StartTagAndEndTag - <myth BOUND= nAMe='john'> ... </myth>
-        BOUND - DoubleQuotes
-            Code span - Gen<None> - [] - ImplicitExpressionEditHandler;Accepts:AnyExceptNewline;ImplicitExpression[ATD];K14 - (12:0,12) - Tokens:0
-        nAMe - SingleQuotes
-            Markup span - Gen<Markup> - [john] - SpanEditHandler;Accepts:Any - (19:0,19) - Tokens:1
-                SyntaxKind.Text;[john];
+RazorDocument - [0..32)::32 - [<myth BOUND= nAMe='john'></myth>]
+    MarkupBlock - [0..32)::32
+        MarkupTagHelperElement - [0..32)::32 - myth[StartTagAndEndTag] - mythTagHelper
+            MarkupTagHelperStartTag - [0..25)::25
+                MarkupTextLiteral - [0..5)::5 - [<myth] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                    OpenAngle;[<];
+                    Text;[myth];
+                MarkupTagHelperAttribute - [5..12)::7 - BOUND - DoubleQuotes - [ BOUND=]
+                    MarkupTextLiteral - [5..6)::1 - [ ] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                        Whitespace;[ ];
+                    MarkupTextLiteral - [6..11)::5 - [BOUND] - Gen<Markup> - SpanEditHandler;Accepts:Any
+                        Text;[BOUND];
+                    Equals;[=];
+                    MarkupTagHelperAttributeValue - [12..12)::0
+                        CSharpExpressionLiteral - [12..12)::0 - [] - Gen<None> - ImplicitExpressionEditHandler;Accepts:AnyExceptNewline;ImplicitExpression[ATD];K14
```

Comment:
Fun fact: We want to generate a placeholder node when the taghelper attribute value is empty. We don't do that for non-taghelper attributes.